cleanup-leftover-volumes/cleanup-elastio-ebs-vols.yaml

```diff
@@ -0,0 +1,98 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: 'AWS::Serverless-2016-10-31'
+Resources:
+  EBSVolumeCleanupFunction:
+    Type: 'AWS::Serverless::Function'
+    Properties:
+      FunctionName: EBSVolumeCleanupFunction
+      Description: 'A Lambda function to delete un-attached EBS volumes older than 24 hours with the elastio:resource tag'
+      Runtime: nodejs16.x
+      Handler: index.handler
+      Role: !GetAtt LambdaExecutionRole.Arn
+      InlineCode: |
+        const AWS = require('aws-sdk');
+        const ec2 = new AWS.EC2();
+
+        const isVolumeOlderThan24Hours = (createTime) => {
+          const currentTime = new Date();
+          const volumeCreateTime = new Date(createTime);
+          const timeDifference = currentTime - volumeCreateTime;
+          const hoursDifference = timeDifference / (1000 * 60 * 60);
+          return hoursDifference > 24;
+        };
+
+        exports.handler = async (event) => {
+          try {
+            const { Volumes } = await ec2.describeVolumes({
+              Filters: [
+                {
+                  Name: 'status',
+                  Values: ['available']
+                },
+                {
+                  Name: 'tag:elastio:resource',
+                  Values: ['*']
+                }
+              ]
+            }).promise();
+
+            for (const volume of Volumes) {
+              if (isVolumeOlderThan24Hours(volume.CreateTime)) {
+                await ec2.deleteVolume({ VolumeId: volume.VolumeId }).promise();
+              }
+            }
+
+            return { statusCode: 200, body: JSON.stringify({ message: 'EBS volumes cleanup successful' }) };
+          } catch (error) {
+            console.error(error);
+            throw error;
+          }
+        };
+      Events:
+        Schedule1:
+          Type: Schedule
+          Properties:
+            Schedule: 'rate(1 hour)'
```

Comment:
Our janitor GC lambda runs every 24 hours. Maybe we should make this lambda daily too, or make janitor lamba hourly instead? I made it low frequency originally, because I assume the cases when we leak volumes are very rare and it will be sufficient to make `janitor` a daily task

Reply:
That's a good point.  Daily should be enough for this too.